src/languageSupport/languages/flux/monaco.flux.syntax.ts

```diff
@@ -1,4 +1,9 @@
 import register from 'src/languageSupport/monaco.onigasm'
+import * as monaco from 'monaco-editor/esm/vs/editor/editor.api'
+
+if (!self.monaco) {
+  self.monaco = monaco
+}
```

Comment:
There is longer a global monaco instance on window.monaco.
https://github.com/TypeFox/monaco-languageclient/pull/248#issuecomment-774026638

I choose to set it explicitly, since we are already utilizing this global object across the application. The global instance is set per language syntax file (toml, markdown, flux), as that file usually gets loaded first.

Reply:
Also note that web workers use `self`, not window. So have to change global used.